docs/inso-cli/cli-command-reference/inso-export-spec.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: article-detail
+title:  inso export spec
+category: "CLI Command Reference"
+category-url: inso-cli
+---
+
+This command will extract and export the raw OpenAPI specification from the data store. If the `--output` option is not specified, the specification will print to console.
+
+### Command
+
+`inso export spec [identifier]`
+
+Export the given specification, the user will be prompted to select a specification if one is not passed as an option. This can also be a file on your filesystem.
```

Comment:
I think this line is redundant given the summary line above? The last part is also incorrect, `identifier` cannot be a file on the filesystem for this command.